lucille-core/src/test/java/com/kmwllc/lucille/core/RunnerTest.java

```diff
@@ -755,32 +755,30 @@ public void testIndexerDeleteMarkerConfig() throws Exception {
   public void testMetrics() throws Exception {
     SharedMetricRegistries.clear();
     assertEquals(0, SharedMetricRegistries.names().size());
-    Runner.runInTestMode("RunnerTest/twoConnectors.conf");
+    Map<String, TestMessenger> runnerMessengerMap = Runner.runInTestMode("RunnerTest/twoConnectors.conf");
+    String connector1ID = runnerMessengerMap.get("connector1").getRunId();
```

Comment:
please rename connector1ID to run1ID. What you are getting here is the runID for the first run we're executing in this test. We don't really have a notion of IDs for connectors (though they can have names specified in the config).

you can add a junit assertion that run1ID is the same as what you are currently calling connector2ID below, and you don't need to store that value separately

after you do the second run (i.e. second call to Runner.runInTestMode) down below, please repeat this logic -- capture the new runId -- call it run2ID -- and make sure that we can access the timers beginning with that second runId -- in the code that looks like  metrics.getTimers(MetricFilter.contains...